Data/icons/map_bridge.svg

```diff
@@ -10,10 +10,10 @@
    xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
    xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
    version="1.1"
-   width="6"
-   height="6"
+   width="5"
+   height="5.0000005"
```

Comment:
these changes seem unrelated to the intent of the pull request 🤔

Reply:
you can delete this, I can't remember this change. Or mean you the complete pull request split all commits in different branches. Ouu I'm beginner in using git.

Reply:
I think, you have to split the pull request, looking the different commits, than the modifications are clearly.

Reply:
This commit can be ignored  
https://github.com/XCSoar/XCSoar/pull/133/commits/1cb85d167c0e65e45b18bb0c431a1ce98a12f8a3